src/plugin/duration/index.js

```diff
@@ -25,8 +24,8 @@ const unitToMS = {
 
 const isDuration = d => d instanceof Duration // eslint-disable-line no-use-before-define
 
-let $d
-let $u
+let $d = null
+let $u = null
```

Comment:
I initialized it to null to explicitly write that the value is currently empty.

Reply:
any reason to initialize it to null, can we simply use `let $d`?

Reply:
@iamkun 
Yes, my initial intention was to explicitly initialize null to indicate that there is no current value, but I'll revert back to that.